prow/pj-on-kind.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Requires go, docker, and kubectl.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Prep and check args.
+job="${1:-""}"
+config="${CONFIG_PATH:-""}"
+job_config="${JOB_CONFIG_PATH:-""}"
+out_dir="${OUT_DIR:-/tmp/prowjob-out}"
+
+echo job=${job}
+echo CONFIG_PATH=${config}
+echo JOB_CONFIG_PATH=${job_config}
+echo OUT_DIR=${out_dir} "(May be different when reusing an existing kind cluster.)"
+
+if [[ -z ${job} ]]
+then
+  echo "Must specify a job name as the first argument."
+  exit 2
+fi
+if [[ -z ${config} ]]
+then
+  echo "Must specify config.yaml location via CONFIG_PATH env var."
+  exit 2
+fi
+if [[ -n ${job_config} ]]
+then
+  job_config="--job-config-path=${job_config}"
+fi
+
+# Install kind and set up cluster if not already done.
+if [[ -z $(which kind) ]]
+then
+  GO111MODULE="on" go get sigs.k8s.io/kind@v0.4.0 && kind create cluster
+fi
+found="false"
+for clust in $(kind get clusters)
+do
+  if [[ ${clust} == "mkpod" ]]
+  then
+    found="true"
+  fi
+done
+if [[ ${found} == "false" ]]
```

Comment:
nit: could just inline this into the `if` where you find the match instead of doing it after the loop`